GameServerApp/GameServerApp.csproj

```diff
@@ -76,7 +76,7 @@
     <_CopyCommand>xcopy /s /y /q /d</_CopyCommand>
   </PropertyGroup>
   <PropertyGroup Condition="'$(OS)' == 'Unix'">
-    <_CopyCommand>cp -r -u</_CopyCommand>
+    <_CopyCommand>cp -r</_CopyCommand>
```

Comment:
Mac wouldn't work with the -u there. Something to do with difference in commands across Unix. There might be a better way to fix it.